### PR TITLE
Zadanie 3

### PR DESCRIPTION
**Pull request by Algiz25**
Zrobiem zadanie 3

### CODE REVIEW

**Review by wsparcie — APPROVED** [review]
fajnie ze zwiekszono num_samples do 10 przy wizualizacjach na koncu, bo o wiele lepiej mozna porownac rozne predykcje
czysta i poprawna implementacja samego u-neta
zabraklo data augmentation w vocresizewrapper (chocby prostego hflip), wiec model dosc szybko uczy sie zdjec na pamiec
ustawienie base=64 w smallunet to duzo jak na taki set, model robi sie ogromny i moze overfittowac
przy loss function (crossentropy) brakuje argumentu class_weights wiec czarne tlo moze zdominowac trening

ogolnie git

**Comment by sz3kz** [comment]
Ogólnie dobrze wykonane zadanie.

Fajnie wygląda w zadaniu od @wsparcie **augmentacja danych** oraz **obniżenie kar za tło**, co moim zdaniem dodatkowo polepszy działanie modelu tutaj.

Bilans `epochs` do `base` prowadzi do troche lepszych masek niż u mnie.

**Comment by Pewciu6** [comment]
LGTM 🚀

**Review by Pewciu6 — APPROVED** [review]

**Review by jakubbaldyga — APPROVED** [review]
Hej ogólnie tak jak wyżej, ale ogólnie spoko
